### PR TITLE
#10148 Copy Update For Homepage

### LINKED ISSUE #10148: Copy Update for Homepage
Make copy updates according to the details in the [Home Page v1 Copy](https://docs.google.com/document/d/1pE3WUcrH0cyv6vcfM5UL-n1Xfqt75ao9mx80c73G3fQ/edit#heading=h.u60md9rmxkia) document

**Home Page v1 Copy**: https://docs.google.com/document/d/1pE3WUcrH0cyv6vcfM5UL-n1Xfqt75ao9mx80c73G3fQ/edit#heading=h.u60md9rmxkia

Original Design Ticket => https://app.zenhub.com/workspaces/web-experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/10021

## Acceptance Criteria
- [ ] Homepage reflects the content updates in the above file
- [ ] Review with Audrey (Content) before deploying live

### PR DESCRIPTION
**Pull request by jpandersen87**
Closes #10148 

This PR updates related components for the home page to reflect content updates. Some obsolete json-based content loading was removed in favor of explicit JSX. The use of the custom `rs-hero` class was removed in favor of USWDS built-ins.

### CODE REVIEW

File: frontend-react/src/pages/home/Sections/Section.tsx [hunk 1]
```diff
@@ -1,12 +1,9 @@
 import { SectionProp } from "../HomeProps";
 
-import CtaSection from "./CtaSection";
 import LiveMapSection from "./LiveMapSection";
 
 export default function Section({ section }: { section: SectionProp }) {
-    if (section.type === "cta") return <CtaSection section={section} />;
-    else if (section.type === "liveMap")
-        return <LiveMapSection section={section} />;
+    if (section.type === "liveMap") return <LiveMapSection section={section} />;
```
Comment (penny-lischer) [inline frontend-react/src/pages/home/Sections/Section.tsx:6]: Should we add the section types to an enum?

Comment (jpandersen87) [reply frontend-react/src/pages/home/Sections/Section.tsx:6]: Negative. The Home page is expected to go through a complete rewrite in the future.

**Review by penny-lischer — APPROVED** [review]
Small nit otherwise LGTM

**Review by etanb — APPROVED** [review]

**Comment by audreykwr** [comment]
Hey Joseph! Just noticed a couple things:

1. Some missing links under the process diagram image: 

> While ReportStream currently works with COVID-19 and mpox data, we are adding more reportable conditions soon. [Contact us](mailto:reportstream@cdc.gov) if you would like updates on additional conditions as they become available. 
> 
> [Learn how to send data through our API](https://reportstream.cdc.gov/resources/api).

2. Additionally about that section, I think the learn how sentence should be a separate paragraph. @jillian-hammer thoughts on that from your perspective?

3. Above the diagram image is my own error - "test results" needs to be deleted. So the content will read: 

> Our open-source platform aggregates and securely delivers health care data from organizations and testing facilities directly to public health entities.

4. The connect with us at the bottom needs to link to reportstream@cdc.gov rather than the contact us page.

Thanks!

File: frontend-react/src/pages/home/Home.tsx [hunk 2]
```diff
@@ -45,6 +46,29 @@ export const Home = () => {
...
+                        Our open-source platform aggregates and securely
+                        delivers health care data test results from
+                        organizations and testing facilities directly to public
+                        health entities.
+                    </p>
+                    <img
+                        src="/assets/img/ReportStreamDiagram_2022.png"
+                        alt="Organizations and testing facilities submit data, which ReportStream validates, augments and standardizes. Then, the data is routed as ReportStream determines the destination by subscription management. After this, it is transformed and transported to public health agencies in the desired data model and batch size."
+                    />
+                    <p>
```
Comment (jillian-hammer) [inline frontend-react/src/pages/home/Home.tsx:63]: @jpandersen87 Can you apply `class="usa-prose"` to this section to improve legibility & consistency with the text styling in the above section?

File: frontend-react/src/pages/home/Home.tsx [hunk 3]
```diff
@@ -45,6 +46,29 @@ export const Home = () => {
...
+                        While ReportStream currently works with COVID-19 and
+                        mpox data, we are adding more reportable conditions
+                        soon. Contact us if you would like updates on additional
+                        conditions as they become available. Learn how to send
```
Comment (jillian-hammer) [inline frontend-react/src/pages/home/Home.tsx:67]: Add a line break between "Contact us if you would like..." and "Learn how to send..." sentences

File: frontend-react/src/pages/home/Home.tsx (hunk 2 above)
Comment (jpandersen87) [reply frontend-react/src/pages/home/Home.tsx:63]: Good eye on noticing what is actually a site-wide issue due to USWDS. USWDS DOES NOT apply site-wide baseline styles. Their `usa-prose` also only applies to direct children elements, and not to all elements inside. We should async/sync on this sometime, as I have plans to create a site-wide override. In the meantime, will def add the `usa-prose` here for this PR.

**Comment by audreykwr** [comment]
Thanks! One note from me - when you bumped "learn more how to send data.." to a new line you left it in the previous paragraph. So remove the duplicate sentence in the first paragraph, like so:

> While ReportStream currently works with COVID-19 and mpox data, we are adding more reportable conditions soon. [Contact us](mailto:reportstream@cdc.gov) if you would like updates on additional conditions as they become available.
> 
> [Learn how to send data through our API](https://reportstream.cdc.gov/resources/api).

**Comment by jpandersen87** [comment]
> Thanks! One note from me - when you bumped "learn more how to send data.." to a new line you left it in the previous paragraph. So remove the duplicate sentence in the first paragraph, like so:
> 
> > While ReportStream currently works with COVID-19 and mpox data, we are adding more reportable conditions soon. [Contact us](mailto:reportstream@cdc.gov) if you would like updates on additional conditions as they become available.
> > [Learn how to send data through our API](https://reportstream.cdc.gov/resources/api).

Fixed, thanks for the callout

**Review by jillian-hammer — APPROVED** [review]
looks great!! TYSM! @jpandersen87

**Pull request merged**